src/decode.h

```diff
@@ -307,3 +300,9 @@ typedef struct rv_insn {
 
 /* decode the RISC-V instruction */
 bool rv_decode(rv_insn_t *ir, const uint32_t insn);
+
+/* Detect the instruction is RV32C or not */
+FORCE_INLINE bool is_compressed(uint32_t insn)
```

Comment:
This type of helper function is mostly concerned with implementation details and is best off not being placed in `src/decode.h` public header.